### PR TITLE
[FEATURE/BUGFIX]: Fix broken price pair endpoints and some minor improvements / deletions.

### PR DESCRIPTION
**Pull request by SeedyROM**
Fix typo, remove unsupported chains (AION and Stellar) and update price pair endpoint URLs.

### CODE REVIEW

File: web3data/handlers/market.py [hunk 1]
```diff
@@ -215,7 +215,7 @@ def order_book(self, pair: str, **kwargs) -> Dict:
             params=kwargs,
         )
 
-    def uniswap_liqudity(self, pair: str, **kwargs) -> Dict:
+    def uniswap_liquidity(self, pair: str, **kwargs) -> Dict:
```
Comment (SeedyROM) [inline web3data/handlers/market.py:218]: Amberdata is going to have a new endpoint coming out soon that will have more info for liquidity and include other exchanges. Consider deprecating this?

Comment (dmuhs) [reply web3data/handlers/market.py:218]: Will the endpoint be phased out at a certain date? I can log a warning and remove it in a new release after it's removed. :slightly_smiling_face:

**Review by dmuhs — APPROVED** [review]

**Comment by dmuhs** [comment]
Thanks for the contribution @SeedyROM! I definitely appreciate the hacking effort over the holidays :tada: 
I'll push it out in the next release soon with some other things I have cooking.

File: web3data/handlers/market.py (hunk 1 above)
Comment (SeedyROM) [reply web3data/handlers/market.py:218]: I think for now we will support it until something comes along to break it. A warning would be nice, but not required.

Sorry for the late reply!

Comment (dmuhs) [reply web3data/handlers/market.py:218]: I added a warning in version `0.1.7` which just landed. 👍🏼 Good suggestion!

**Pull request merged**